### PR TITLE
fix: app selector item image styles

### PR DESCRIPTION
**Pull request by enesozturk**
# Description

A slight update to the image styles of the app selector item.

# Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Additional Information (Optional)

Slack conversation: https://walletconnect.slack.com/archives/C044SKFKELR/p1704384062573449

### CODE REVIEW

File: index.html
Comment (devceline) [inline index.html:1]: Why change this file here for app selector styles?

**Review by devceline — COMMENTED** [review]
Looks like some changes from the PWA branch are here

**Comment by enesozturk** [comment]
@devceline should be fixed now, since on the mobile, the images have different sizes, previous value doesn't meet requirements for fully rounded image

**Comment by chris13524** [comment]
I think still broken?

image

**Comment by enesozturk** [comment]
@chris13524 Did you try with the recent preview link? https://app-web3inbox-r0a1vn87u-walletconnect1.vercel.app/

Cannot reproduce it with my device (iOS 17), I'm thinking if `rem` is not responding as expected in some devices, I'll need to check in some simulators, don't have Android device

**Comment by chris13524** [comment]
@enesozturk the current build looks good.

I don't understand why we have multiple border radiuses though, we should just have 1 and then e.g. `overflow: hidden;` and clip the content

**Review by chris13524 — APPROVED** [review]

**Comment by enesozturk** [comment]
@chris13524 

We use `box-shadow` instead of `border` to give inset (inside) borders since CSS doesn't have inset border feature. 

Another problem is that you cannot use inside shadows for an element that renders the image inside. So the `overflow` approach doesn't work here. 

```jsx
<div> /* Using inset shadow doesn't work since the image overrides it. */
 <img/>
</div>
```

Instead, we're using `::after` CSS selector to be able to set `box-shadow` over an image.

**Pull request merged**